vimiv/imutils/metadata.py

```diff
@@ -0,0 +1,284 @@
+# vim: ft=python fileencoding=utf-8 sw=4 et sts=4
+
+# This file is part of vimiv.
+# Copyright 2017-2023 Christian Karl (karlch) <karlch at protonmail dot com>
+# License: GNU GPL v3, see the "LICENSE" and "AUTHORS" files for details.
+
+"""Utility functions and classes for metadata handling.
+
+This module provides a common interface for all metadata related functionalities.
+`MetadataHandler` is used to interact with the metadata of the current image. It
+relies on the functionality provided by optionally loaded metadata plugins. Such
+plugins implements the `MetadataPlugin` abstract class and registers that class
+using the `register` function.
+
+Module Attributes:
+    _registry: List of registered `MetadataPlugin` implementations.
+"""
+
+from abc import ABC, abstractmethod
```

Comment:
Please use `import abc` for consistency :blush:

Reply:
Changed in f5d29bd4b6fe5a993dcd4f0a85232773abafa7fc